utils/index.d.ts

```diff
@@ -880,6 +880,55 @@ export interface IWizardOptions<T extends IActionContext> {
     showLoadingPrompt?: boolean;
 }
 
+export interface ActivityTreeItemOptions {
+    label: string;
+    contextValuePostfix?: string;
+    collapsibleState?: TreeItemCollapsibleState;
+    children?: (parent: AzExtParentTreeItem) => AzExtTreeItem[];
+}
+
+type ActivityEventData<T> = ActivityTreeItemOptions & T;
+
+export type OnStartActivityData = ActivityEventData<{}>;
```

Comment:
Do we need to have a type for each activity event?  VS Code seems to just have all the `onEvents` have the typing of an `Event<any>` or `Event<void>`

Reply:
The specific types are for the event datas, I wrote it like this with the intention on making it easy to add things to the data of each event later if needed.

Reply:
@nturinski I'm gonna leave the event data names as they are... I like them

Reply:
Do you as you will.... kids these days just don't understand grammar. 🧓

Reply:
Well to me, it's the data for the "OnStart" event. I guess I think of the event being "OnStart" rather than just "Start"

Reply:
> Do you as you will.... kids these days just don't understand grammar. 🧓

Ok boomer :wink: